extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -319,13 +325,14 @@ export class Bind {
       match = this.compareStringArrays_(initialValue, classes);
     } else {
       const attribute = element.getAttribute(property);
+      initialValue = attribute;
       if (typeof expectedValue === 'boolean') {
-        initialValue = !!attribute;
+        // Boolean attributes return values of either '' or null.
+        match = (expectedValue && initialValue === '')
```

Comment:
IMO this is probably not a common use case and muddles the API a bit. Also, the expression would have to be `[attr]="'true'"`, which evaluates to a string.